src/deploy/campaign/get-config.ts

```diff
@@ -68,10 +68,20 @@ export const getConfig = async ({
   governors ?: Array<string>;
   admins ?: Array<string>;
   zeroVaultAddress ?: string;
-  env ?: string;
-}) : Promise<IZNSCampaignConfig<SignerWithAddress | Wallet>> => {
+  env ?: TEnvironment;
+}) : Promise<IZNSCampaignConfig> => {
+  // Prioritize reading from the env variable first, and only then fallback to the param
+  let envLevel = process.env.ENV_LEVEL as TEnvironment;
```

Comment:
Could simplify this and any other null checks with `const envLevel = env ?? process.env.ENV_LEVEL;` or the opposite order.

Also this seems a bit odd because it will read `ENV_LEVEL` but it will always fallback to `env` if specified, but from your comment it sounds like it should only fallback to `env` if the env var wasn't found in the system with `process.env.ENV_LEVEL`